packages/runtime-plugin-hmr/src/index.ts

```diff
@@ -6,13 +6,13 @@ import { applyHotUpdates, createHotContext } from './hot-module-state';
 import { HmrUpdatePacket, HmrUpdateResult } from './types';
 
 // TODO using host and port from the config, default to use location.host
-const port = 9801;
+const port = HMR_PORT;
```

Comment:
This piece should be taken from the user config, not written dead.

Reply:
yes , it can be configed by user now！ ![image](https://user-images.githubusercontent.com/51783403/233523237-2f457dff-9bc0-436d-a5c3-298505dc8508.png)
![image](https://user-images.githubusercontent.com/51783403/233523247-471dd852-1a21-44db-aeba-795fca37b359.png)

Reply:
If you open two services at the same time, there will be a problem， This should be judged together with the variables of defaultConfig and define, and port conflicts should be prevented.

Reply:
it association #105

Reply:
soga ,  what should i do ?  🙂

Reply:
> If you open two services at the same time, there will be a problem， This should be judged together with the variables of defaultConfig and define, and port conflicts should be prevented.

predecessor  open two services at the same time 。How should I understand

Reply:
https://discord.com/channels/1088114165505327104/1088114166033817603

Reply:
I mean, you can write this for the time being, but if you run example and vue demo at the same time, the websocket ports will conflict.

Reply:
老哥 哪个 DIscord 线上交流的 能再发一下吗  晚上我有问题 想问问你 现在在打工

Reply:
> I mean, you can write this for the time being, but if you run example and vue demo at the same time, the websocket ports will conflict.

老哥 哪个 DIscord 线上交流的 能再发一下吗 晚上我有问题 想问问你 现在在打工

Reply:
加微信吧 y1256029807

Reply:
ok